csaf/parser.py

```diff
@@ -195,6 +195,12 @@ def _process_vulnerabilities(self):
             if "product_status" in vulnerability:
                 for product_status in vulnerability["product_status"]:
                     vuln_info.set_value("status", product_status)
+                    if "known_affected" in vulnerability["product_status"]:
+                        affected_product_ids = []
+                        known_affected = vulnerability["product_status"].get("known_affected", [])
+                        for affected_product_id in known_affected:
+                            affected_product_ids.append(affected_product_id)
+                        vuln_info.set_value("affected_product_ids", affected_product_ids )
```

Comment:
just to have the naming similar to the CSAF file

```suggestion
                        vuln_info.set_value("known_affected_product_ids", affected_product_ids )
```